modules/utils/rewriteBareModuleIdentifiers.js

```diff
@@ -23,7 +23,8 @@ export default function rewriteBareModuleIdentifiers(code, packageConfig) {
     plugins: [
       unpkgRewrite(origin, dependencies),
       '@babel/plugin-proposal-optional-chaining',
-      '@babel/plugin-proposal-nullish-coalescing-operator'
+      '@babel/plugin-proposal-nullish-coalescing-operator',
+      '@babel/plugin-proposal-class-properties'
```

Comment:
Need to add to https://github.com/mjackson/unpkg/blob/master/package.json

Reply:
both of those are already included in package.json